debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/Filters.java

```diff
@@ -62,17 +78,20 @@ protected static boolean isNotBuiltInTable(TableId id) {
     private final Predicate<String> isBuiltInDb;
     private final Predicate<TableId> isBuiltInTable;
     private final ColumnNameFilter columnFilter;
+    private final Predicate<TableId> isIgnoredTable;
 
     private Filters(Predicate<String> dbFilter,
                     Predicate<TableId> tableFilter,
                     Predicate<String> isBuiltInDb,
                     Predicate<TableId> isBuiltInTable,
-                    ColumnNameFilter columnFilter) {
+                    ColumnNameFilter columnFilter,
+                    Predicate<TableId> isIgnoredTable) {
```

Comment:
Tiny nit: could swap the last two params, so that the predicates stay together?

Reply:
sure.